src/solvers/strings/string_builtin_function.cpp

```diff
@@ -426,8 +426,8 @@ string_constraintst string_insertion_builtin_functiont::constraints(
 {
   if(args.size() == 1)
   {
-    auto pair = add_axioms_for_insert(
-      generator.fresh_symbol, result, input1, input2, args[0], array_pool);
+    auto pair = generator.add_axioms_for_insert(
+      result, input1, input2, args[0]);
```

Comment:
this commit should be squashed in the previous one if it doesn't compile without it